server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -789,6 +791,63 @@ func extractOptionsFromStartedBuildEvent(event *build_event_stream.BuildEvent) (
 	return "", nil
 }
 
+func getCursesLinesFromStartedBuildEvent(event *build_event_stream.BuildEvent) int {
+	// the number of lines curses can overwrite is 2 + the ui_actions shown:
```

Comment:
nit: getCursesLines -> getNumCursesLines so it's clear that we are returning the number of lines, not lines

Reply:
Yup, np.